Mage.Sets/src/mage/cards/e/EyeOfMalcator.java

```diff
@@ -0,0 +1,43 @@
+package mage.cards.e;
+
+import mage.abilities.common.EntersBattlefieldAbility;
+import mage.abilities.common.EntersBattlefieldAllTriggeredAbility;
+import mage.abilities.common.EntersBattlefieldControlledTriggeredAbility;
+import mage.abilities.effects.common.continuous.BecomesCreatureSourceEffect;
+import mage.abilities.effects.keyword.ScryEffect;
+import mage.cards.Card;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.SubType;
+import mage.filter.StaticFilters;
+import mage.game.permanent.token.custom.CreatureToken;
+
+import java.util.UUID;
+
+public class EyeOfMalcator extends CardImpl {
+    public EyeOfMalcator(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ARTIFACT}, "{2}{U}");
+
+        //When Eye of Malcator enters the battlefield, scry 2.
+        this.addAbility(new EntersBattlefieldAbility(new ScryEffect(2)));
```

Comment:
This needs to use an instance of `EntersBattlefieldTriggeredAbility`, not `EntersBattlefieldAbility`

Reply:
done